File: Day8/day08.py
try: 
    with open("/workspaces/AdventOfCode-2025/Day8/data", 'r') as file:
        file = file.readlines()
        for index1, coords in enumerate(file):
            for index2, coords2 in enumerate(file):
                if index2 <= index1:
                    continue
                    
                
                print(index1, index2)




except FileNotFoundError:
    print("not found")

File: Day8/day08.java
package Day8;

import java.io.File;
import java.io.FileNotFoundException;
import java.util.ArrayList;
import java.util.Scanner;

public class day08 {
    public static void main(String[] args) {
        ArrayList<String> fileData = getFileData("/workspaces/AdventOfCode-2025/Day8/data");
        ArrayList<ArrayList<String>> coords = new ArrayList<ArrayList<String>>();
        for (String cords : fileData) {
            coords.add(cords.split(","));
        }
    }

    public static ArrayList<String> getFileData(String fileName) {
        ArrayList<String> fileData = new ArrayList<String>();
        try {
            File f = new File(fileName);
            Scanner s = new Scanner(f);
            while (s.hasNextLine()) {
                String line = s.nextLine();
                if (!line.equals(""))
                    fileData.add(line);
            }
            return fileData;
        }
        catch (FileNotFoundException e) {
            return fileData;
        }
    }
}
